package.json

```diff
@@ -9,6 +9,7 @@
   "scripts": {
     "start": "ember server",
     "build": "ember build",
+    "postinstall": "bower install",
```

Comment:
Hah--I obviously didn't notice it had been there and yanked before. :-) I
can revert that particular commit in the morning.
On Jan 27, 2015 7:52 PM, "Toran Billups" <notifications@github.com> wrote:

> In package.json
> <https://github.com/nchristus/ember-cli-password-toggle/pull/5#discussion_r23660511>
> :
>
> > @@ -9,6 +9,7 @@
> >    "scripts": {
> >      "start": "ember server",
> >      "build": "ember build",
> > +    "postinstall": "bower install",
>
> Just a quick note here - at first this looks like a win/win for people
> maintaining the library but it actually hurts the ember-cli consumer app
> during install (and was removed in a previous PR if you look at the commit
> history). Could you pull this from the PR before we merge it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nchristus/ember-cli-password-toggle/pull/5/files#r23660511>
> .
>